src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -49,18 +49,51 @@
       <DataSource Persistence="ProjectFile" PersistedName="Win32Resource" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringListProperty Name="AvailablePlatforms" Separator="," Visible="False" />
-  <StringProperty Name="FileName" DisplayName="Project File" ReadOnly="True">
+    <StringProperty Name="DefineConstants" DisplayName="Define Constants"/>
+    <EnumProperty Name="PlatformTarget" DisplayName="Platform Target"/>
+    <StringProperty Name="Prefer32Bit" DisplayName="Prefer 32Bit"/>
+    <StringProperty Name="AllowUnsafeBlocks"  Default="False"  DisplayName="Allow unsafe code"/>
+    <StringProperty Name="Optimize" DisplayName="Optimize"/>
+    <EnumProperty Name="WarningLevel" DisplayName="Warning Level"/>
+    <StringProperty Name="NoWarn" DisplayName="Supress Warning"/>
+    <BoolProperty Name="TreatWarningsAsErrors" Visible="True"  Default="False" Description="Treat warnings as errors" />
+    <StringProperty Name="OutputPath" DisplayName="Output Path"/>
+    <StringProperty Name="DocumentationFile" DisplayName="Documentation file"/>
+    <EnumProperty Name="GenerateSerializationAssemblies" DisplayName="Generate serialization assemblies">
+        <EnumValue Name="Auto" DisplayName="Auto" IsDefault="True" />
+        <EnumValue Name="On" DisplayName="On" />
+        <EnumValue Name="Off" DisplayName="Off" />
+    </EnumProperty>
+    <EnumProperty Name="LanguageVersion" DisplayName="Language version"/>
+    <EnumProperty Name="ErrorReport" DisplayName="Error report"/>
+    <EnumProperty Name="DebugInfo" DisplayName="Debug Info">
+        <EnumProperty.DataSource>
+            <DataSource Persistence="ProjectFile" PersistedName="DebugType" />
+        </EnumProperty.DataSource>
+    </EnumProperty>
+    <StringProperty Name="DebugSymbols" DisplayName="Debug symbols"/>
+    <EnumProperty Name="FileAlignment" DisplayName="File Alignment"/>
+    <StringProperty Name="BaseAddress" DisplayName="Base address" />
+    <StringProperty Name="PreBuildEvent" DisplayName="Pre Build Event"/>
+    <StringProperty Name="PostBuildEvent" DisplayName="Post Build Event"/>
+    <EnumProperty Name="RunPostBuildEvent" DisplayName="Run Post Build Event">
+        <EnumValue Name="Always" DisplayName="Always" />
+        <EnumValue Name="OnBuildSuccess" DisplayName="On successful build"  IsDefault="True" />
+        <EnumValue Name="OnOutputUpdated" DisplayName="When the build updates the project output" />
+    </EnumProperty>
+    <StringProperty Name="ReferencePath" DisplayName="Reference Path"/>
+    <StringListProperty Name="AvailablePlatforms" Separator="," Visible="False" />
```

Comment:
Why is this Visible=False? Does it show up in the browseobject if visible=false?